example/App.js

```diff
@@ -87,19 +85,21 @@ export default class App extends React.Component {
 
   onButtonPress = () => {
     Fidel.start();
-  }
+  };
 
-  handleError = (error) => {
-    console.log("Error message: " + error.message);
+  handleError = error => {
+    console.log('Error message: ' + error.message);
     switch (error.type) {
       case Fidel.ErrorType.userCanceled:
-        console.log("User canceled the process");
+        console.log('User canceled the process');
         break;
       case Fidel.ErrorType.sdkConfigurationError:
-        console.log("Please configure the Fidel SDK correctly");
+        console.log('Please configure the Fidel SDK correctly');
```

Comment:
Out of curiosity, when do we use double " and when '?

Reply:
My prettier automatically did it by saving the file, but I have already reverted the changes 👍